Bottom: e5c30dde4c292d3cfa5d71faca70f8a4b7c6a657
Top:    ee0f8cc03d6be387bbec3ca0b3297ea6b0f51bd7
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 17:43:11 +0000

Refresh of refactor-polygon-copy

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 19ec771..c1861d8 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2983,7 +2983,6 @@ poly_Copy1 (POLYAREA * dst, const POLYAREA * src)
       r_insert_entry (dst->contour_tree, (BoxType *) * last, 0);
       last = &(*last)->next;
     }
-
   return TRUE;
 }
